samples/wifi/twt/README.rst

```diff
@@ -22,6 +22,21 @@ Overview
 The sample can perform Wi-Fi operations such as connect and disconnect in the 2.4 GHz and 5 GHz bands depending on the capabilities of an access point.
 
 Using this sample, the development kit can connect to the specified access point in :abbr:`STA (Station)` mode and setup TWT flow with the access point.
+By default once the TWT flow is setup, the sample will use the traffic generator module to send and receive TCP data packets to and from the traffic generator
+server running on a PC connected to the same access point (typically via Ethernet).
```

Comment:
```suggestion
server running on a PC connected to the same access point (typically through Ethernet).
```
Please also combine this part with the link above (so the same sentence is all on the same line).